integration_tests/Cargo.toml

```diff
@@ -805,6 +809,7 @@ all = [
 	"sts",
 	"support",
 	"swf",
+	"timestream-query",
```

Comment:
You need to add `timestream-write` here too?

Reply:
Added